pvlib/clearsky.py

```diff
@@ -756,6 +760,14 @@ def rmse(alpha):
         components['slope_max'] = c5
         components['mean_nan'] = c6
         components['windows'] = clear_windows
+
+        components['mean_diff_array'] = np.abs(meas_mean - alpha * clear_mean)
+        components['max_diff_array'] = np.abs(meas_max - alpha * clear_max)
+        components['line_length_array'] = meas_line_length - clear_line_length
+        components['slope_nstd_array'] = meas_slope_nstd
+        components['slope_max_array'] = (np.max(
```

Comment:
maybe "values" instead of "array"?

Reply:
`_flag` on the previous set of keys, removed `_array` here

Reply:
I like the change, but it would break user code if people are calling `return_components`. Probably not likely to be a serious issue but we should think twice about doing this. In any case it should be listed as an API change.

Reply:
what was meant by "values" instead of "array"?  I'm ok reverting this change, but I thought you were requesting renaming the keys.

Reply:
Sorry for not being clear. I originally meant to suggest only slightly different names for the new keys that you've added. I was not suggesting renaming the existing keys. I thought "array" was an undesirable identifier because everything is an array here.

I do like the latest proposal, but we'd need to document it as an API change.